AlphaWallet/TokenScriptClient/Models/XMLHandler.swift

```diff
@@ -800,7 +783,7 @@ extension XMLHandler {
     }
 
     fileprivate static func getServerForNativeCurrencyAction(fromRoot root: XMLDocument, xmlContext: XmlContext) -> RPCServer? {
-        return root.at_xpath("/action/input/token/ethereum".addToXPath(namespacePrefix: xmlContext.namespacePrefix), namespaces: xmlContext.namespaces)?["network"].flatMap { Int($0) }.flatMap { RPCServer(chainID: $0) }
+        return root.at_xpath("/card/input/token/ethereum:transfer".addToXPath(namespacePrefix: xmlContext.namespacePrefix), namespaces: xmlContext.namespaces)?["network"].flatMap { Int($0) }.flatMap { RPCServer(chainID: $0) }
```

Comment:
This wouldn't work because `addToXPath()` breaks up the path components and adds the prefix `xmlContext.namespacePrefix` to each of them. `addToXPath()` is used because adding the prefix to each component in that string literal was getting untidy.

So probably have to do this instead (untested):

```
let path = "/card/input/token".addToXPath(namespacePrefix: xmlContext.namespacePrefix) + "/ethereum:transfer"
return root.at_xpath(path, namespaces: xmlContext.namespaces)?["network"].flatMap { Int($0) }.flatMap { RPCServer(chainID: $0) }
```

Reply:
But I don't think this line should be changed. It's still "ethereum", not "ethereum:transfer" because this is used for action cards, like this:

```
<ts:input>
    <ts:token name="xdai">
        <ts:ethereum network="100"/>
    </ts:token>
</ts:input>
```

Reply:
So the change for "ethereum:transfer" has to be made somewhere else

Reply:
this has changed again to ethereum:value so I am removing the above change and keeping it as is (value is still called value but with the ethereum: prefix rather than ts: prefix, the xmlcontext should get this I believe)